roboteam_ai/test/SkillTests/KickTest.cpp

```diff
@@ -58,7 +58,7 @@ TEST(KickTest, It_sends_proper_robotcommands) {
 }
 
 TEST(KickTest, It_chips) {
-    ros::Rate rate(1);
+    ros::Rate rate(60);
```

Comment:
Why change this?

Reply:
I changed this in more places, mostly because this is also the rate our system runs at and because it means we have faster tests.

 I suspect the rate is also related to issue #66, but I had trouble reproducing that problem, so cannot say anything with certainty there.

Reply:
Yes. that is related. ROS may need some time to start up the advertising/subscribing; that's why the rate was so low